tapdb/assets_store.go

```diff
@@ -873,12 +873,38 @@ func (a *AssetStore) constraintsToDbFilter(
 				Valid: true,
 			}
 		}
+
+		if query.MaxAmt != 0 {
+			assetFilter.MaxAmt = sql.NullInt64{
+				Int64: int64(query.MaxAmt),
+				Valid: true,
+			}
+		}
+
 		if query.MinAnchorHeight != 0 {
 			assetFilter.MinAnchorHeight = sqlInt32(
 				query.MinAnchorHeight,
 			)
 		}
 
+		if query.ScriptKey != nil {
+			key := query.ScriptKey.PubKey
+			assetFilter.TweakedScriptKey = key.SerializeCompressed()
+		}
+
+		if query.AnchorPoint != nil {
+			anchorPointBytes, err := encodeOutpoint(
+				*query.AnchorPoint,
+			)
+			if err != nil {
+				return QueryAssetFilters{}, fmt.Errorf(
+					"unable to encode outpoint: %w", err,
+				)
```

Comment:
nit: calls to `Errorf()` can have the closing parenthesis on the same line as the arguments (see formatting exception in the formatting doc).